op-dispute-mon/mon/extract/output_agreement_enricher_test.go

```diff
@@ -79,39 +79,48 @@ func TestDetector_CheckOutputRootAgreement(t *testing.T) {
 		}
 	})
 
-	t.Run("OutputFetchFails", func(t *testing.T) {
-		validator, rollup, metrics := setupOutputValidatorTest(t)
-		rollup.outputErr = errors.New("boom")
+	t.Run("AllNodesReturnError", func(t *testing.T) {
+		validator, clients, metrics := setupMultiNodeTest(t, 3)
+		for _, client := range clients {
+			client.outputErr = errors.New("boom")
+		}
 		game := &types.EnrichedGameData{
 			L1HeadNum:     100,
 			L2BlockNumber: 0,
 			RootClaim:     mockRootClaim,
 		}
 		err := validator.Enrich(context.Background(), rpcblock.Latest, nil, game)
-		require.ErrorIs(t, err, rollup.outputErr)
+		require.Error(t, err)
+		require.Contains(t, err.Error(), "all nodes returned errors")
```

Comment:
Let's introduce a `ErrAllNodesUnavailable` variable that we return when all nodes fail and then we can check for it explicitly with `ErrorIs` here.

Reply:
Done.